client/src/ipfsUploader.js

```diff
@@ -0,0 +1,13 @@
+import { create } from 'ipfs-http-client'
+
+export const uploadToIpfs = async (file) => {
+    const client = create('https://localhost:5001')
```

Comment:
where's port `5001` configured?

Reply:
What I'm concerned is: will this port change due to version update?

When we initiate ipfs node, is it always gonna be port 5001, can we specify the port 5001 to make sure this is the case?

Reply:
I'm still exploring IPFS :)
I will definitely check how to set the port number to make it more robust, seems fragile, good catch.